src/lib/translators/mongo.ts

```diff
@@ -309,7 +309,10 @@ function transformAggregate(step: Readonly<S.AggregationStep>): MongoStep[] {
   const idblock: PropMap<string> = columnMap(step.on);
   const group: { [id: string]: {} } = {};
   const project: PropMap<any> = {};
+  let ret: MongoStep[] = [];
```

Comment:
This let is not needed, just return at the end of the if/else block.
At the very least rename it `returned`, `result` or `steps`.

Reply:
👍